lib/secretsscanner/scaner/scan.go

```diff
@@ -250,3 +274,19 @@ func privateKeyNameGen(path, deviceID, fingerprint string) string {
 	sha.Write([]byte(fingerprint))
 	return hex.EncodeToString(sha.Sum(nil))
 }
+
+// expandSkipDirs expands the glob patterns in the skipDirs list and returns a set of the
+// directories matched by the glob patterns to be skipped.
+func expandSkipDirs(skipDirs []string) (map[string]struct{}, error) {
+	skippedDirs := make(map[string]struct{})
+	for _, glob := range skipDirs {
```

Comment:
There's nothing making skipDirs target only directories, right? Should we call it skipPaths and let it apply to files too?

Reply:
Yeah, that's correct but my idea was to skip only directories. Added support for files as well